Documentation/GettingStarted.md

```diff
@@ -54,9 +57,112 @@ Add a user to the libary. The user can be of any class, as long the calls implem
         NSLog(@"Failed to create Account: %@", error);
     }
 ```
+### 4. Inbound calls
 
-#### 4. Set callback for incoming calls
+#### 1. AppDelegate - instantiate CallKitProviderDelegate to a private property 
```

Comment:
There is no need to make it public because only the delegate will interact with it. Setting it up with the incoming call block could also be a solution, but we think that it is more clear the way we do it in the example app. Ofcourse could everyone come up with a better solution, we just did what worked for us.